1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -1,3 +1,28 @@
+// display(pickFromArray(quotes));
+function pickFromArray(quotes) {
+  return quotes[(Math.random() * quotes.length)] ;
+}
+let texes = document.getElementById("textQuote");
+let authors = document.getElementById("nameAuthor");
+let button = document.getElementById("new");
+
+button.addEventListener("click", () => {
+  let getQuotesEl = pickFromArray(quotes);
+  texes.innerText = `${getQuotesEl.quote}`;
+  authors.innerText = `${getQuotesEl.author}`;
+});
```

Comment:
Nice one. You could even make the code that displays the quote (text and author) as a separate little helper function. Care with naming` let getQuotesEl ` suggests an element.  Its just a quote selected at random. So perhaps

`let randomQuote = pickFromArray(quotes); `